app/components/home/transaction-list/transaction-list-context-menu.ts

```diff
@@ -0,0 +1,90 @@
+import { remote } from 'electron';
+import { useClipboard } from '@blockstack/ui';
+import { Transaction } from '@blockstack/stacks-blockchain-sidecar-types';
+
+import { hasMemo, getRecipientAddress } from '../../../utils/tx-utils';
+
+export function registerHandler(el: HTMLDivElement | null, handler: (e: Event) => void) {
+  if (el === null) return;
+  el.addEventListener('contextmenu', handler, false);
```

Comment:
for perf we should always default to [passive event listeners](https://web.dev/uses-passive-event-listeners/), esp because this is always going to be in chromium.

Reply:
it says this is only really beneficial for touch/mouse events? Does no harm to add though.